api/libs/webshot/src/webshot.dto.ts

```diff
@@ -34,26 +30,16 @@ export class WebshotPngConfig extends WebshotConfig {
   screenshotOptions?: ScreenshotOptions;
 }
 
-export class WebshotPdfConfig extends WebshotConfig {
+export class WebshotBasicPdfConfig extends WebshotConfig {
   @ApiPropertyOptional()
   @IsOptional()
   pdfOptions?: PDFOptions;
-
-  @ValidateNested()
-  @IsObject()
-  @Type(() => ReportOptions)
-  @ApiProperty()
-  reportOptions!: ReportOptions;
 }
 
-export class WebshotSFComparisonMapPdfConfig extends WebshotConfig {
-  @ApiPropertyOptional()
-  @IsOptional()
-  pdfOptions?: PDFOptions;
-
+export class WebshotPdfReportConfig extends WebshotBasicPdfConfig {
```

Comment:
I miss multiple inheritance 😬 
But it's ok. I don't think it's worth to use mixins here just for these tiny data structures.

Reply:
fixed :)

Reply:
this was genuinely a wish related to easier composition in TS.

but I also like the way you changed this to make things simpler to follow 😄